model/pdata/common_alias.go

```diff
@@ -48,11 +48,26 @@ var (
 	NewAttributeValueBytes  = pdata.NewAttributeValueBytes
 )
 
-// AttributeMap is an alias for pdata.AttributeMap struct.
-type AttributeMap = pdata.AttributeMap
+// Map is an alias for pdata.Map struct.
+type Map = pdata.Map
 
-// Aliases for functions to create pdata.AttributeMap.
+// Deprecated: [v0.47.0] Use Map instead.
+type AttributeMap = pdata.Map
+
+// Aliases for functions to create pdata.Map.
 var (
-	NewAttributeMap        = pdata.NewAttributeMap
-	NewAttributeMapFromMap = pdata.NewAttributeMapFromMap
+	NewMap        = pdata.NewMap
+	NewMapFromRaw = pdata.NewMapFromRaw
```

Comment:
nit: separate PR, I would prefer to rename `internal/pdata` to maybe just `internal`, I am confused about the 2 packages named the same, when code refers to one vs the other.